src/NUnitEngine/nunit.engine/Internal/ProcessModel.cs

```diff
@@ -37,7 +39,12 @@ public enum ProcessModel
         /// <summary>
         /// Run tests directly in the NUnit process
         /// </summary>
-        Single,
+        InProcess,
+        /// <summary>
+        /// Run tests directly in the NUnit process
+        /// </summary>
+        [Obsolete("Use InProcess instead")]
+        Single = InProcess,
```

Comment:
Doing code reviews pays off, I didn't know you could assign another enum value to make two equivalent.